doc/source/serve/deploy_model_tutorial.rst

```diff
@@ -0,0 +1,329 @@
+to access a model that summarizes text. Let's take a look at how it works.
+This is the code for the model:
+
+.. literalinclude:: ../../../python/ray/serve/examples/doc/e2e_local.py
+  :language: python
+  :start-after: __local_model_start__
+  :end-before: __local_model_end__
+
+The Python file, called ``local_model.py`` uses the ``summarize`` function to
+generate summaries of text. The ``summarizer`` variable inside ``summarize``
+points to a function that uses the
+`t5-small <https://huggingface.co/t5-small>`_ model to summarize text.
+When ``summarizer`` is called on a Python String, it returns summarized text
+encapsulated within a dictionary and a list. ``summarize`` then removes the
+dictionary and list, and it returns only the summary. The file can be run
+locally by executing the Python script, which uses the model to summarize an
+article about the Apollo 11 moon landing.
+
+.. code-block:: bash
+
+  $ python local_model.py
+
+  "two astronauts steered their fragile lunar module safely and smoothly to the
+  historic landing . the first men to reach the moon -- Armstrong and his
+  co-pilot, col. Edwin E. Aldrin Jr. of the air force -- brought their ship to
+  rest on a level, rock-strewn plain ."
+
+This tutorial's goal is to deploy this model using Ray Serve, so it can be
+queried over HTTP. We'll start by converting the above Python function into a
+Ray Serve deployment that can be launched locally on a laptop.
+
+We start by opening a new Python file. First, we need to import ``ray`` and
+``ray serve``, to use features in Ray Serve such as ``deployments``, which
+provide HTTP access to our model.
+
+.. literalinclude:: ../../../python/ray/serve/examples/doc/e2e_deployment.py
+    :language: python
+    :start-after: __import_start__
+    :end-before: __import_end__
+
+After these imports, we can include our model code from above. 
+We won't call our ``summarize`` function just yet though! 
+We will soon add logic to handle HTTP requests, so the ``summarize`` function 
+can operate on article text sent via HTTP request.
+
+.. literalinclude:: ../../../python/ray/serve/examples/doc/e2e_deployment.py
+  :language: python
+  :start-after: __local_model_start__
+  :end-before: __local_model_end__
+
+Ray Serve needs to run on top of a Ray Cluster, so we connect to a local one:
+
+.. literalinclude:: ../../../python/ray/serve/examples/doc/e2e_deployment.py
+  :language: python
+  :start-after: __start_ray_cluster_start__
+  :end-before: __start_ray_cluster_end__
+
+The ``address`` parameter in ``ray.init()`` connects your Serve script to a
+running local Ray cluster. Later, we'll discuss how to start a local Ray
+cluster. The ``namespace`` parameter determines what Ray objects are accessible
+in this script. Setting it to ``serve`` ensures that the Ray Serve objects we
+create are accessible by default.
```

Comment:
I would not mention namespaces here to avoid introducing an unnecessary new concept